org.eclipse.january.examples/src/org/eclipse/january/examples/dataset/NumpyExamples.java

```diff
@@ -565,7 +570,7 @@ public void logicalAnd() {
     	
     	// Equivalent tech
     	BooleanDataset bs = Comparisons.logicalAnd(a, b);
-      	System.out.println("Result of a && b "+bs);
+      	System.out.println("Result of a & b "+bs);
```

Comment:
While I understand why you have done & instead of && here, logical and being & normally. The documentation of Comparisons.logicalAnd is documented as returning the && of a and b. 

So for this change to be accepted we would need the Comparisons.logicalAnd javadocs to be updated to match. 

However I am not sure the change is appropriate. The implementation does indeed do a short circuit && 

It is therefore probably more appropriate to document the short circuiting if it is a guarantee, or leave it as is if not.